cwl_utils/sandboxjs.py

```diff
@@ -317,17 +319,21 @@ def new_js_proc(
                             [container_engine, "images", "-q", nodeimg],
                             universal_newlines=True,
                         )
+                    elif container_engine == "singularity":
+                        try:
+                            singularityimgs = glob.glob(os.environ['CWL_SINGULARITY_CACHE'] + '/node_slim.sif')
+                        except KeyError:
+                            singularityimgs = glob.glob(os.getcwd() + '/node_slim.sif')
```

Comment:
I think it is better to avoid throwing exceptions in normal program flow. You could replace the try-except part with:
```
singularity_cache = os.environ.get('CWL_SINGULARITY_CACHE')
if singularity_cache is None:
  singularityimgs = glob.glob(os.getcwd() + '/node_slim.sif')
else
  singularityimgs = glob.glob(singularity_cache + '/node_slim.sif')
```